data/data/bootstrap/files/usr/local/bin/bootstrap-cluster-gather.sh

```diff
@@ -0,0 +1,77 @@
+    queue resources/api-pods oc --request-timeout=5s get pods -l apiserver=true --all-namespaces --template '{{ range .items }}-n {{ .metadata.namespace }} {{ .metadata.name }}{{ "\n" }}{{ end }}'
+
+    queue resources/apiservices.json oc --request-timeout=5s get apiservices -o json
+    queue resources/clusteroperators.json oc --request-timeout=5s get clusteroperators -o json
+    queue resources/clusterversion.json oc --request-timeout=5s get clusterversion -o json
+    queue resources/configmaps.json oc --request-timeout=5s get configmaps --all-namespaces -o json
+    queue resources/csr.json oc --request-timeout=5s get csr -o json
+    queue resources/endpoints.json oc --request-timeout=5s get endpoints --all-namespaces -o json
+    queue resources/events.json oc --request-timeout=5s get events --all-namespaces -o json
+    queue resources/kubeapiserver.json oc --request-timeout=5s get kubeapiserver -o json
+    queue resources/kubecontrollermanager.json oc --request-timeout=5s get kubecontrollermanager -o json
+    queue resources/machineconfigpools.json oc --request-timeout=5s get machineconfigpools -o json
+    queue resources/machineconfigs.json oc --request-timeout=5s get machineconfigs -o json
+    queue resources/namespaces.json oc --request-timeout=5s get namespaces -o json
+    queue resources/nodes.json oc --request-timeout=5s get nodes -o json
+    queue resources/openshiftapiserver.json oc --request-timeout=5s get openshiftapiserver -o json
+    queue resources/pods.json oc --request-timeout=5s get pods --all-namespaces -o json
+    queue resources/rolebindings.json oc --request-timeout=5s get rolebindings --all-namespaces -o json
+    queue resources/roles.json oc --request-timeout=5s get roles --all-namespaces -o json
+    # this just lists names and number of keys
+    queue resources/secrets-names.txt oc --request-timeout=5s get secrets --all-namespaces
+    # this adds annotations, but strips out the SA tokens and dockercfg secrets which are noisy and may contain secrets in the annotations
+    queue resources/secrets-names-with-annotations.txt oc --request-timeout=5s get secrets --all-namespaces -o=custom-columns=NAMESPACE:.metadata.namespace,NAME:.metadata.name,TYPE:.type,ANNOTATIONS:.metadata.annotations | grep -v -- '-token-' | grep -v -- '-dockercfg-'
+    queue resources/services.json oc --request-timeout=5s get services --all-namespaces -o json
+
+    FILTER=gzip queue resources/openapi.json.gz oc --request-timeout=5s get --raw /openapi/v2
+
+    echo "Waiting for logs ..."
+    wait
+
+    if (( $(stat -c%s '/tmp/artifacts-api-server-temp/resources/openapi.json.gz') <= 20 ))
+	then
+		rm -rf "/tmp/artifacts-api-server-temp"
+	else
+		rm -rf "${HOME}/artifacts-api-server-temp/resources"
+		mv "/tmp/artifacts-api-server-temp/resources" "${HOME}/artifacts-api-server-temp"
```

Comment:
We need a `mkdir` here, too, right?
```suggestion
		mkdir -p "${HOME}/artifacts-api-server-temp"
		mv "/tmp/artifacts-api-server-temp/resources" "${HOME}/artifacts-api-server-temp"
```